openapi/components/schemas/CheckoutForm/CommonCheckoutForm.yaml

```
@@ -0,0 +1,118 @@
+type: object
+required:
+  - websiteId
+  - plans
+properties:
+  id:
+    description: The checkout form identifier string
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  websiteId:
+    description: Checkout form website ID
+    type: string
+    allOf:
+      - $ref: ../ResourceId.yaml
+  plans:
+    description: >-
+      The list of plans that would be applied to customer order by default.
+    minItems: 1
+    allOf:
+      - $ref: CheckoutFormPlans.yaml
+  addonPlans:
+    description: >-
+      The list of independent addons.
+
+      Customer will be able to select whether to add the plans from the list or not.
+    default: []
+    allOf:
+      - $ref: CheckoutFormPlans.yaml
+  bumpPlans:
+    description: >-
+      The list of plans that can be used instead of ones specified in `plans`.
+
+      Customer will be able to select whether to purchase bump plans or the regular ones.
+    nullable: true
+    default: null
+    minItems: 1
+    allOf:
+      - $ref: CheckoutFormPlans.yaml
+  accountsEnabled:
+    description: If `true` customer will be able to sign-up and sign-in using the form
+    type: boolean
+    default: false
+  couponsEnabled:
+    description: If `true` customer will be able to use coupons on the form
+    type: boolean
+    default: false
+  purchaseLimit:
+    description: >-
+      If the purchase limit is set – every purchase will decrease it. Zero
+      purchase limit will make form inactive
```

End every description with proper punctuation.
```suggestion
      purchase limit will make form inactive.
```